Products/CMFPlone/resources/browser/combine.py

```diff
@@ -0,0 +1,103 @@
+from datetime import datetime
+from plone.registry.interfaces import IRegistry
+from plone.resource.file import FilesystemFile
+from plone.resource.interfaces import IResourceDirectory
+from Products.CMFPlone.interfaces import IBundleRegistry
+from Products.CMFPlone.interfaces.resources import (
+    OVERRIDE_RESOURCE_DIRECTORY_NAME,
+)
+from StringIO import StringIO
+from zope.component import getUtility
+
+PRODUCTION_RESOURCE_DIRECTORY = "production"
+
+
+def get_production_resource_directory():
+    persistent_directory = getUtility(IResourceDirectory, name="persistent")
+    container = persistent_directory[OVERRIDE_RESOURCE_DIRECTORY_NAME]
+    production_folder = container[PRODUCTION_RESOURCE_DIRECTORY]
+    timestamp = production_folder.readFile('timestamp.txt')
+    return "%s/++unique++%s" % (
+        PRODUCTION_RESOURCE_DIRECTORY, timestamp)
+
+
+def get_resource(context, path):
+    try:
+        resource = context.unrestrictedTraverse(path)
+    except AttributeError:
+        # in testing context, we cannot traverse
+        # we just return fake content
+        return '/* Content of %s */' % path
+    if isinstance(resource, FilesystemFile):
+        (directory, sep, filename) = path.rpartition('/')
+        return context.unrestrictedTraverse(directory).readFile(filename)
+    else:
+        try:
+            return resource.GET()
+        except:
+            return resource()
+
```

Comment:
Done https://github.com/plone/Products.CMFPlone/commit/dba346005111ca999edf6f7148b2cf075f79080b
I have also replace the brutal try/except with a if/else.